plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergRegisterTableProcedure.java

```diff
@@ -80,28 +81,28 @@ protected QueryRunner createQueryRunner()
                 .build();
     }
 
-    @BeforeClass
+    @BeforeAll
     public void initFileSystem()
     {
         fileSystem = getFileSystemFactory(getDistributedQueryRunner()).create(SESSION);
     }
 
-    @AfterClass(alwaysRun = true)
+    @AfterAll
     public void tearDown()
             throws IOException
     {
         deleteRecursively(metastoreDir.toPath(), ALLOW_INSECURE);
     }
 
-    @DataProvider
     public static Object[][] fileFormats()
     {
         return Stream.of(IcebergFileFormat.values())
                 .map(icebergFileFormat -> new Object[] {icebergFileFormat})
                 .toArray(Object[][]::new);
     }
 
-    @Test(dataProvider = "fileFormats")
+    @ParameterizedTest
+    @MethodSource("fileFormats")
```

Comment:
We could use `@EnumSource(IcebergFileFormat.class)` instead in this class & TestIcebergMigrateProcedure.